tools/localazy/README.md

```diff
@@ -2,12 +2,35 @@
 
 Localazy is used to host the source strings and their translations.
 
+<!--- TOC -->
+
+* [Localazy project](#localazy-project)
+  * [Key naming rules](#key-naming-rules)
+* [CLI Installation](#cli-installation)
+* [Download translations](#download-translations)
+* [Add translations to a specific module](#add-translations-to-a-specific-module)
+
+<!--- END -->
+
 ## Localazy project
 
-To add new strings, or to translate existing strings, go the the Localazy project: [https://localazy.com/p/element](https://localazy.com/p/element).
+To add new strings, or to translate existing strings, go the the Localazy project: [https://localazy.com/p/element](https://localazy.com/p/element). Please follow the key naming rules (see below).
 
 Never edit manually the files `localazy.xml` or `translations.xml`!.
 
+### Key naming rules
+
+For code clarity and in order to download strings to the correct module, here are some naming rules to follow as much as possible:
```

Comment:
It's not clear to me which of the rules are required for Android's module system and which are merely for consistency.

Also more importantly, I'm not sure if the `screen_...` prefixing will work great for web? It seems that in many cases things that are a screen on mobile would just be one of several views on the same "screen" on web.

Reply:
Thanks for the feedback. For now Localazy is limited to the mobile applications, and I am not sure it will be used on Element Web. But yes, strictly speaking, a `screen` can be a part of a `screen`. We may have the same configuration on mobile OSes, for tablet mode, with the list/detail pattern: 2 screens displayed at the same time. There will be no issue.